base_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './nameable'
+
+class BaseDecorator < Nameable
+  def correct_name?
+    @nameable.correct_name?
```

Comment:
- [ ] As requested in the project requirement, let's make sure this class has a constructor with a namable object from params and is assigned to an instance variable.

hint: 
def initialize(nameable)
 @nameable = nameable
end